source/standalone/environments/state_machine/lift_cube_sm.py

```diff
@@ -13,6 +13,10 @@
 
     ./isaaclab.sh -p source/standalone/environments/state_machine/lift_cube_sm.py --num_envs 32
 
+    or
```

Comment:
You can make a separate state machine for this task.

Reply:
Ok no problem, may I also create a new environment for this task?